src/server/server.js

```diff
@@ -1,8 +1,61 @@
 var express = require('express');
+var db = require('./db');
+var bodyParser = require('body-parser');
 var app = express();
+
+var urlParser = bodyParser.urlencoded({extended: false});
+var running = false;
+var server;
+
+db.create('meatpocalypse.db');
+
+app.post('/index.html', urlParser, (req, res, next) => {
+    if (!req.body)
+        return res.sendStatus(400);
+    var user = req.body.user;
+    var pass = req.body.pass;
+    var msg  = 'Invalid request.';
+
+    if (!user || !pass) {
+        res.writeHead(400, {'Content-Type': 'text/html'});
+        res.write('You must enter a username and a password');
+        res.end();
+    }
+
+    db.getPlayer(user, (err, row) => {
```

Comment:
An interesting side note, this can also be written as:

```javascript
    db.getPlayer(user, (err, row) => {
      // get Player validation
      try {
        if(err) throw err;
        if (row === undefined) {
          throw new Error('Username ' + user + ' does not exist.');
        }
        if(row.pass !== pass) {
          throw new Error("Invalid password.");
        }
        // get player
        res.redirect(302, 'game.html');
        return res.end();

      } catch(ex) {
        // handle errors
        res.writeHead(400, {'Content-Type': 'text/html'});
        res.write(ex.message); // send the customized error message
        res.end();
      }
    });

```